db/warehouse_structure.sql

```diff
@@ -21401,6 +21452,18 @@ CREATE SEQUENCE public.temp_export_client_100s_id_seq
     CACHE 1;
 
 
+--
+-- Name: temp_export_client_101s_id_seq; Type: SEQUENCE; Schema: public; Owner: -
```

Comment:
Hm. These are all the `temp_` tables that will get removed from this file next time Zach or I make a PR :\

Reply:
Hmm, so those are actually sequences, not tables.  I'm not sure where they came from, but I can see if I can delete them from my local DB.  They shouldn't be doing anything useful.